examples/rules/data/stories.md

```diff
@@ -1,73 +1,58 @@
 <!-- each story will be perceived as independent rule -->
 
-
-## activate loop q_form
+>> activate loop q_form
 <!-- required slots for q_form should be listed somewhere else -->
     - ...
 * activate_q_form  <!-- like request_restaurant -->
-    - action_activate_q_form  <!-- default action with an utterance like action_restart -->
-    - form{"name": "q_form"} <!-- problem with predicting this event, because, it has to be used as condition -->
-    - action_loop_q_form  <!-- can be anything -->
+    - loop_q_form  <!-- Activate form -->
 
-## ask loop q_form
-<!-- RulePolicy should substitute requested_slot with actual slot value -->
-    - form{"name": "q_form"}  <!-- condition that form is active-->
-    - slot{"requested_slot": "some_slot"}  <!-- some condition -->
-    - ...
-* inform{"some_slot":"bla"} <!-- can be ANY -->
-    - validate_some_slot
-    - action_loop_q_form <!-- can be internal core action, can be anything -->
 
-## explain loop q_form
-    - form{"name": "q_form"} <!-- condition that form is active-->
+>> explain loop q_form (unhappy path)
+    - form{"name": "loop_q_form"} <!-- condition that form is active-->
     - slot{"requested_slot": "some_slot"}  <!-- some condition -->
     - ...
 * explain                          <!-- can be anything -->
     - utter_explain_some_slot
-    - action_loop_q_form
+    - loop_q_form
 
-## stop loop q_form
-    - form{"name": "q_form"} <!-- condition that form is active-->
-    - ...
-* stopp
-    - action_stop_q_form
 
-## finish loop q_form
-    - form{"name": "q_form"} <!-- condition that form is active-->
-    - slot{"requested_slot": null} <!-- some condition to finish -->
+>> submit form
     - ...
-    - action_stop_q_form
+    - loop_q_form
```

Comment:
we need a condition that the form was active above, because `loop_q_form` will be perceived as a rule as well as `utter_stop`

Reply:
Why are they perceived as two independent rules?

Reply:
> we need a condition that the form was active above

`loop_q_form` should be enough as condition, no? This action can only run when the form was active.